src/slicer/SlicingBox.js

```diff
@@ -71,15 +73,32 @@ export default class SlicingBox extends SlicingToolBase {
       });
     this.slicerArrows.show();
 
+    const boxPositions = [
+      this.bbox.corners.bottomRight,
+      this.bbox.corners.bottomLeft,
+      this.bbox.corners.topLeft,
+      this.bbox.corners.topRight,
+      this.bbox.corners.bottomRight
+    ];
+    const scratchZeroCartesian2 = Cartesian2.ZERO;
+    const scratchCartesian2 = new Cartesian2(1, 0);
     this.slicingBoxEntity = this.dataSource.entities.add({
       position: new CallbackProperty(() => this.boxCenter, false),
-      orientation: this.bbox.orientation,
-      box: {
-        dimensions: new CallbackProperty(() => new Cartesian3(this.bbox.length, this.bbox.width, this.bbox.height), false),
-        material: SLICING_GEOMETRY_COLOR.withAlpha(0.1),
-        outline: true,
-        outlineColor: SLICING_GEOMETRY_COLOR,
-      },
+      polylineVolume: {
+        positions: new CallbackProperty(() => {
+          const height = Cartographic.fromCartesian(this.bbox.center).height - (this.bbox.height / 2);
+          return updateHeightForCartesianPositions(boxPositions, height);
+        }, false),
+        cornerType: CornerType.MITERED,
+        outline: false,
+        material: SLICING_GEOMETRY_COLOR.withAlpha(0.2),
+        shape: new CallbackProperty(() => [
+          scratchZeroCartesian2,
+          scratchZeroCartesian2,
+          scratchCartesian2,
+          new Cartesian2(0, this.bbox.height),
```

Comment:
You should use a scratch for `new Cartesian2(0, this.bbox.height)`, not for `Cartesian2.ZERO`.